assets/js/scripts.js

```diff
@@ -527,161 +544,162 @@ $("#language").on("change", function () {
+document.querySelector('.menu-toggle').addEventListener('click', function () {
+  document.querySelector('.side-menu').classList.toggle('menu-opened');
+  Settings.isMenuOpened = document.querySelector('.side-menu').classList.contains('menu-opened');
 
-  $('.menu-toggle').text(Settings.isMenuOpened ? 'X' : '>');
+  document.querySelector('.menu-toggle').textContent = Settings.isMenuOpened ? 'X' : '>';
 
-  $('.top-widget').toggleClass('top-widget-menu-opened', Settings.isMenuOpened);
-  $('#fme-container').toggleClass('fme-menu-opened', Settings.isMenuOpened);
+  document.querySelector('.top-widget').classList.toggle('top-widget-menu-opened', Settings.isMenuOpened);
+  document.getElementById('fme-container').classList.toggle('fme-menu-opened', Settings.isMenuOpened);
 });
 
-$('#tooltip-map').on('change', function () {
-  Settings.showTooltipsMap = $('#tooltip-map').prop('checked');
+document.getElementById('tooltip-map').addEventListener('change', function () {
+  Settings.showTooltipsMap = this.checked;
   MapBase.updateTippy('tooltip');
 });
 
-$('#marker-cluster').on("change", function () {
-  Settings.isMarkerClusterEnabled = $("#marker-cluster").prop('checked');
+document.getElementById('marker-cluster').addEventListener('change', function () {
+  Settings.isMarkerClusterEnabled = this.checked;
   MapBase.addMarkers();
 });
 
-$('#enable-marker-popups').on("change", function () {
-  Settings.isPopupsEnabled = $("#enable-marker-popups").prop('checked');
+document.getElementById('enable-marker-popups').addEventListener('change', function () {
+  Settings.isPopupsEnabled = this.checked;
   MapBase.addMarkers();
 });
 
-$('#enable-marker-popups-hover').on("change", function () {
-  Settings.isPopupsHoverEnabled = $("#enable-marker-popups-hover").prop('checked');
+document.getElementById('enable-marker-popups-hover').addEventListener('change', function () {
+  Settings.isPopupsHoverEnabled = this.checked;
 });
 
-$('#enable-marker-shadows').on("change", function () {
-  Settings.isShadowsEnabled = $("#enable-marker-shadows").prop('checked');
+document.getElementById('enable-marker-shadows').addEventListener('change', function () {
+  Settings.isShadowsEnabled = this.checked;
   Treasure.onSettingsChanged();
   Legendary.onSettingsChanged();
   MapBase.addMarkers();
 });
 
-$("#enable-legendary-backgrounds").on("change", function () {
-  Settings.isLaBgEnabled = $("#enable-legendary-backgrounds").prop('checked');
+document.getElementById('enable-legendary-backgrounds').addEventListener('change', function () {
+  Settings.isLaBgEnabled = this.checked;
   Legendary.onSettingsChanged();
 });
 
-$("#legendary-animal-marker-type").on("change", function () {
-  Settings.legendarySpawnIconType = $("#legendary-animal-marker-type").val();
+document.getElementById('invisible-removed-markers').addEventListener('change', function () {
+  Settings.legendarySpawnIconType = this.value;
   Legendary.onSettingsChanged();
 });

```

Comment:
Something went wrong here. This setting does not belong here.
Correct selector is:

```javascript
document.getElementById('legendary-animal-marker-type')
```